src/streamlink/plugins/vaughnlive.py

```diff
@@ -1,17 +1,27 @@
 import re
-
+ 
 from streamlink import StreamError
 from streamlink.plugin import Plugin
-from streamlink.plugin.api import useragents
+from streamlink.plugin.api import http, validate, useragents
 from streamlink.stream import HTTPStream, StreamIOIterWrapper, StreamIOThreadWrapper
+ 
+#_url_re = re.compile(r"""
+#   http(s)?://(\w+\.)?
+#   (?P<domain>breakers|instagib|vapers|pearltime).tv
+#   (/embed/video)?
+#   /(?P<channel>[^/&?]+)
+#""", re.VERBOSE)
 
 _url_re = re.compile(r"""
-    http(s)?://(\w+\.)?
-    (?P<domain>vaughnlive|breakers|instagib|vapers|pearltime).tv
-    (/embed/video)?
-    /(?P<channel>[^/&?]+)
+   http(s)?://(\w+\.)?
+   (?P<domain>vaughn).live
```

Comment:
you need to add some tests

https://github.com/streamlink/streamlink/blob/master/tests/plugins/test_vaughnlive.py

---

update https://github.com/streamlink/streamlink/blob/master/docs/plugin_matrix.rst

---

also the support for all the other domains is missing now